tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
+    <ProductVersion>8.1.0</ProductVersion>
```

Comment:
Why do we need to hard code this? Why can't it be generated from the other MSBuild properties?

Reply:
I tried locally using major and minor version but I think this project didn't import the right props and due to timing just wanted to unblock the build. End to End tests are failing now, so I guess now I have to fix than anyway.

Reply:
Looks like there are several assumptions that the testing infrastructure makes around package versions which may not be correct as they assume that the same package version will be used everywhere, which will certainly not always be the case. I just spent some time trying to fix another package testing issue I was seeing, but couldn't get the versions to flow correctly so due to time I just pushed another change which hardcodes the versions for now to unblock the build and be able to get a build out for validation. We should of course undo these hard coded hacks, but we should also fix these assumptions on versions. cc: @radical

Reply:
Do we have an issue tracking this to fix it in `main`?